From ab7b9878773c5aff0558398b867899576e8f9430 Mon Sep 17 00:00:00 2001
From: KyleWong <kang.wang1988@gmail.com>
Date: Thu, 8 Nov 2018 15:53:04 +0800
Subject: [PATCH] Remove GL_EXT_debug_marker related logic as it won't take
 effect in release build. As to debug&profile mode, it might only effect some
 devices when profiling OpenGL&OpenGL ES, if and only if those unsupported
 functions is called.

---
 src/gpu/gl/GrGLInterface.cpp | 8 --------
 1 file changed, 8 deletions(-)

diff --git a/src/gpu/gl/GrGLInterface.cpp b/src/gpu/gl/GrGLInterface.cpp
index 10f0bbe8a6..e228836341 100644
--- a/src/gpu/gl/GrGLInterface.cpp
+++ b/src/gpu/gl/GrGLInterface.cpp
@@ -393,14 +393,6 @@ bool GrGLInterface::validate() const {
         }
     }
 
-    if (fExtensions.has("GL_EXT_debug_marker")) {
-        if (!fFunctions.fInsertEventMarker ||
-            !fFunctions.fPushGroupMarker ||
-            !fFunctions.fPopGroupMarker) {
-            RETURN_FALSE_INTERFACE
-        }
-    }
-
     if ((kGL_GrGLStandard == fStandard && glVer >= GR_GL_VER(4,3)) ||
         fExtensions.has("GL_ARB_invalidate_subdata")) {
         if (!fFunctions.fInvalidateBufferData ||
-- 
2.17.1 (Apple Git-112)

